common/enablers/enableEpochsHandler.go

```diff
@@ -39,6 +41,330 @@ func (handler *enableEpochsHandler) EpochConfirmed(epoch uint32, _ uint64) {
+		core.ChangeDelegationOwnerFlag:
+		return epoch >= handler.enableEpochsConfig.ESDTMetadataContinuousCleanupEnableEpoch
+	case core.DisableExecByCallerFlag:
+		return epoch >= handler.enableEpochsConfig.DisableExecByCallerEnableEpoch
+	case core.RefactorContextFlag:
+		return epoch >= handler.enableEpochsConfig.RefactorContextEnableEpoch
+	case core.CheckFunctionArgumentFlag:
+		return epoch >= handler.enableEpochsConfig.CheckFunctionArgumentEnableEpoch
+	case core.CheckExecuteOnReadOnlyFlag:
+		return epoch >= handler.enableEpochsConfig.CheckExecuteOnReadOnlyEnableEpoch
+	case core.SetSenderInEeiOutputTransferFlag:
+		return epoch >= handler.enableEpochsConfig.SetSenderInEeiOutputTransferEnableEpoch
+	case core.RefactorPeersMiniBlocksFlag:
+		return epoch >= handler.enableEpochsConfig.RefactorPeersMiniBlocksEnableEpoch
+	case core.SCProcessorV2Flag:
+		return epoch >= handler.enableEpochsConfig.SCProcessorV2EnableEpoch
+	case core.FixAsyncCallBackArgsListFlag:
+		return epoch >= handler.enableEpochsConfig.FixAsyncCallBackArgsListEnableEpoch
+	case core.FixOldTokenLiquidityFlag:
+		return epoch >= handler.enableEpochsConfig.FixOldTokenLiquidityEnableEpoch
+	case core.RuntimeMemStoreLimitFlag:
+		return epoch >= handler.enableEpochsConfig.RuntimeMemStoreLimitEnableEpoch
+	case core.RuntimeCodeSizeFixFlag:
+		return epoch >= handler.enableEpochsConfig.RuntimeCodeSizeFixEnableEpoch
+	case core.MaxBlockchainHookCountersFlag:
+		return epoch >= handler.enableEpochsConfig.MaxBlockchainHookCountersEnableEpoch
+	case core.WipeSingleNFTLiquidityDecreaseFlag:
+		return epoch >= handler.enableEpochsConfig.WipeSingleNFTLiquidityDecreaseEnableEpoch
+	case core.AlwaysSaveTokenMetaDataFlag:
+		return epoch >= handler.enableEpochsConfig.AlwaysSaveTokenMetaDataEnableEpoch
+	case core.SetGuardianFlag:
+		return epoch >= handler.enableEpochsConfig.SetGuardianEnableEpoch
+	case core.RelayedNonceFixFlag:
+		return epoch >= handler.enableEpochsConfig.RelayedNonceFixEnableEpoch
+	case core.ConsistentTokensValuesLengthCheckFlag:
+		return epoch >= handler.enableEpochsConfig.ConsistentTokensValuesLengthCheckEnableEpoch
+	case core.KeepExecOrderOnCreatedSCRsFlag:
+		return epoch >= handler.enableEpochsConfig.KeepExecOrderOnCreatedSCRsEnableEpoch
+	case core.MultiClaimOnDelegationFlag:
+		return epoch >= handler.enableEpochsConfig.MultiClaimOnDelegationEnableEpoch
+	case core.ChangeUsernameFlag:
+		return epoch >= handler.enableEpochsConfig.ChangeUsernameEnableEpoch
+	case core.AutoBalanceDataTriesFlag:
+		return epoch >= handler.enableEpochsConfig.AutoBalanceDataTriesEnableEpoch
+	case core.FixDelegationChangeOwnerOnAccountFlag:
+		return epoch >= handler.enableEpochsConfig.FixDelegationChangeOwnerOnAccountEnableEpoch
+	case core.FixOOGReturnCodeFlag:
+		return epoch >= handler.enableEpochsConfig.FixOOGReturnCodeEnableEpoch
+	case core.DeterministicSortOnValidatorsInfoFixFlag:
+		return epoch >= handler.enableEpochsConfig.DeterministicSortOnValidatorsInfoEnableEpoch
```

Comment:
missing check for `DynamicGasCostForDataTrieStorageLoadEnableEpoch` ? (might need a new update branch operation)
```GO
handler.setFlagValue(epoch >= handler.enableEpochsConfig.DynamicGasCostForDataTrieStorageLoadEnableEpoch, handler.dynamicGasCostForDataTrieStorageLoadFlag, "dynamicGasCostForDataTrieStorageLoadFlag", epoch, handler.enableEpochsConfig.DynamicGasCostForDataTrieStorageLoadEnableEpoch)
```

Reply:
indeed this one is missing due to a missing update branch.. added TODO to be fixed after merge from rc/v1.6.0